src/main/java/com/google/devtools/build/lib/starlarkbuildapi/StarlarkRuleContextApi.java

```diff
@@ -557,8 +557,10 @@ String expandLocation(
               @ParamType(type = Depset.class, generic1 = SymlinkEntryApi.class)
             },
             doc =
-                "Either a SymlinkEntry depset or the map of symlinks, prefixed by workspace name,"
-                    + " to be added to the runfiles. See <a"
+                "Either a SymlinkEntry depset or the map of symlinks to be added to the runfiles."
+                    + " Symlinks are always added under the main workspace's runfiles directory"
+                    + " (e.g. <code>&lt;runfiles_root>/_main/&lt;symlink_path></code>, <b>not</b>" +
+                    " the directory corresponding to the current target's repository. See <a"
```

Comment:
could we also update the docs? they're in [//site/en/extending/rules.md](https://github.com/bazelbuild/bazel/blob/master/site/en/extending/rules.md)

Reply:
Done!